### ISSUE TITLE
Base versification off of UGNT & UHB (not ULT)

### ISSUE THREAD

**Issue by benjore** [unfoldingWord/translationCore#5866]
### User Story

As a [type of user], I want [some goal] so that [some reason].

### Features / Specifications

- [ ] What would it take to pivot all of the versification in tC to the Original Languages (UGNT for the NT and UHB for the OT)?
- [ ] Note: There may be one exception to this - In wA, If there is an extra verse in the target language, that might need to show up in the side menu so that the words can be accessed and moved to a different verse for alignment purposes. 
- [ ] Update the verse mapping when UGNT or UHB is updated

### Definition of Done

- [ ] Assumption: There will be less churn if the versification is based on the original languages than any GL. 

#### Work needed (from spike https://github.com/unfoldingword-dev/translationcore/issues/5886):
- [ ] missing verse check  - **_change MissingVersesHelpers.findMissingVerses() to call getExpectedBookVerses() using OrigL (it defaults to en_ult)_**
- [ ] Project open - **_BibleHelpers.isProjectMissingVerses() using 'en_ult`_**
- [ ] Export USFM - **_WordAlignmentHelpers.convertAlignmentDataToUSFM() using 'en_ult`_**
- [ ] tCore group menu initialization for wA tool:
  - **_groupDataHelpers.generateChapterGroupData() uses en_ult, change to use superset of OrigL and targetL chapters/verses_**
  - **_in tCore, ResourcesHelpers.getResourcesNeededByTool() - adds en_ult for Expanded SP, can be removed once Expanded SP fixed._**

- [ ] expanded scripture pane
  - **_ExpandedScripturePaneModal/ChapterView/index.js/render uses en_ul, change to use superset of OrigL and targetL verses_**

**Comment by benjore** [comment]
Blocked by spike

**Comment by benjore** [comment]
Assumption: "Disputed" verses will be added into the UGNT delineated with `[[ ]]`. 

Specification: tC needs to be smart enough to NOT flag those "disputed" verses as missing (i.e. during the project import process).

**Comment by PhotoNomad0** [comment]
@cckozie - do you have an example of a Bible that I could test this with?  Maybe a bible book that has verse counts that are different between ugnt and en_ult?  Or maybe a project that has verses not in en_ult or ugnt?

**Comment by cckozie** [comment]
@PhotoNomad0 - Unfortunately I don't have any examples but suspect Larry would know of some.

**Comment by PhotoNomad0** [comment]
**Note:** putting this on hold till we determine if this is needed at this time or should be reduced in scope to only dealing with extra verses in target language.

**Comment by jag3773** [comment]
Long term we need a general way to handle different versification systems.  Moving from the ULT to the UHB and UGNT is a step in the right direction, but it does not achieve the bigger picture by itself.  Currently all of our resources are using the exact same versification system, this was the simplest way to ensure there weren't discrepancies, so the ULT matches what our UHB and UGNT have.

Presumably we need to use a versification system as the base to map too, or we could come up with some other (unique id?) system to map too. From this standpoint, it probably doesn't matter what system is the base as long as everything else can define its relationship to the base.

If we implement this now for the quarter one release and we also make changes to the word alignment tool then we could unlock the feature of aligning disputed verses (arguably, even said feature doesn't require switching to UHB/UGNT...). Other than that we don't get any user features out of this change. My preference would be to wait until we handle versification differences systemically in quarter two.

Up to you @birchamp in terms of timing, however.

**Comment by jag3773** [comment]
@birchamp I do think that adding support for https://github.com/Copenhagen-Alliance/versification-specification/tree/master/versification-mappings is the right direction to go.

I think that the user story here needs defined well.  e.g. as it is titled there isn't really any benefit to the user, it's just a technical swap with no value.  If this issue is to support any versification system so that a user can check their translation when it doesn't match our resources then that seems valuable.

**Comment by BincyJ** [comment]
@birchamp to confirm if possibility of RCL handling versification approach
